insteon_mqtt/device/FanLinc.py

```diff
@@ -498,11 +498,12 @@ def link_data_from_pretty(self, is_controller, data):
         if not is_controller:
             if 'group' in data:
                 data_3 = data['group']
-                if 'ramp' in data and data['group'] <= 0x01:
-                    data_2 = 0x1f
-                    for ramp_key, ramp_value in Dimmer.ramp_pretty:
-                        if data['ramp'] >= ramp_value:
-                            data_2 = ramp_key
+            if 'ramp_rate' in data and (data_3 is None or data_3 == 0x01):
```

Comment:
I think this should still be `data_3 <= 0x01` as there may be some links that are for group 0x00 that should still get the default value too.

Reply:
I agree.  In particular, if there are any FanLinc devices that use group 0 for the dimmer, then we wouldn't want to break those.

Will change as suggested here.

Reply:
Fixed in commit 497071cf29c042f88b2dad63aee5c799e0c0c58c.

All scene tests still pass.  Smoke tested on my system with a dry-run of the sync-all command.